### PR TITLE
notes drawer styles

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
Address the following issues:
https://github.com/bldrs-ai/Share/issues/304
part of this issue
https://github.com/bldrs-ai/Share/issues/306

Adjusted:
- Notes panel NavBar
- Note card

### CODE REVIEW

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter please take a look

**Comment by pablo-mayrgundter** [comment]
Looking better!  There's a problem with paging.. load KNIK and then page issues.. you can get to 3/4 but not to 4/4.

**Comment by pablo-mayrgundter** [comment]
Mobile (iPhone12) looks to have too much padding..

image

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
/docs directory shouldn't be part of this or future PRs.  Did you sync to head to create this branch?  Might want to restart it..

**Comment by pablo-mayrgundter** [comment]
This looks like it's pulling in Markus's proposal to remove metadata.

Could you list in the description which issues you're addressing with this PR?

I don't agree with that change.. I think it's right for smth like Portfolio mode when we have that, but in general I think it helps orient the user that these are indeed comments to see the usual metadata.

**Comment by pablo-mayrgundter** [comment]
How about omiting the pager when there's only a single issue? 
image

**Comment by OlegMoshkovich** [comment]
Address the following issues:
https://github.com/bldrs-ai/Share/issues/304
part of this issue
https://github.com/bldrs-ai/Share/issues/306
part of this issue
https://github.com/bldrs-ai/Share/issues/305

**Comment by pablo-mayrgundter** [comment]
Did left padding grow?  I thought this was left-aligned with "Notes" before.. 
image

**Comment by OlegMoshkovich** [comment]
image

image

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter PTAL

File: src/Components/IssueCard.test.jsx [hunk 1]
```diff
@@ -19,7 +19,6 @@ test('IssueCard', () => {
       </ShareMock>)
   expect(screen.getByText('new_title')).toBeInTheDocument()
   expect(screen.getByText('2000-01-01 00:00:00Z')).toBeInTheDocument()
-  expect(screen.getByText('bob')).toBeInTheDocument()
```
Comment (pablo-mayrgundter) [inline src/Components/IssueCard.test.jsx:22]: Shouldn't this be included?

File: src/Components/IssuesControl.jsx [hunk 2]
```diff
@@ -56,42 +56,45 @@ export function IssuesNavBar() {
...
+        {selectedIssueId && issues.length > 1 &&
+          <>
+            <TooltipIconButton
+              title='Previous Note'
+              placement='bottom'
+              size='small'
+              onClick={() => selectIssue('previous')}
+              icon={<PreviousIcon style={{width: '20px', height: '20px'}}/>}
+            />
+            {/* <div style={{fontSize: '10px'}}> {selectedIssueIndex + 1} of {issues.length}</div> */}
```
Comment (pablo-mayrgundter) [inline src/Components/IssuesControl.jsx:84]: remove

**Review by pablo-mayrgundter — APPROVED** [review]
LGTM. 

One change for unit tests.. looks like you removed the assert for username in previous change, but should be back now.  Added comment below...

**Pull request merged**